content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -706,7 +709,47 @@ To benefit from this speedup, all these conditions must be met:
 For any other volume types, SELinux relabelling happens another way: the container
 runtime  recursively changes the SELinux label for all inodes (files and directories)
 in the volume.
-The more files and directories in the volume, the longer that relabelling takes.
+
+{{< feature-state feature_gate_name="SELinuxChangePolicy" >}}
+For Pods that want to opt-out from relabeling using mount options, they can set
+`spec.securityContext.seLinuxChangePolicy` to `Recursive`. This is required
+when multiple pods share a single volume on the same node, but they run with
+different SELinux labels that allows simultaneous access to the volume. For example, a privileged pod
+running with label `spc_t` and an unprivileged pod running with the default label `container_file_t`.
+With unset `spec.securityContext.seLinuxChangePolicy` (or with the value `MountOption`),
+only one of such pods is able to run on a node, the other one gets ContainerCreating with error
+`conflicting SELinux labels of volume <name of the volume>: <label of the running pod> and <label of the pod that can't start>`.
+
+#### SELinuxWarningController
+To make it easier to identify Pods that are affected by the change in SELinux volume relabeling,
+a new controller called `SELinuxWarningController` has been introduced in kube-controller-manager.
+It is disabled by default and can be enabled by setting the `--controllers=*,selinux-warning-controller` command line flag
+and `SELinuxChangePolicy` feature gate.
+When enabled, the controller observes running Pods and when it detects that two Pods use the same volume
+with different SELinux labels:
+1. It emits an event to both of the Pods. `kubectl describe pod <pod-name>` the shows
+  `SELinuxLabel "<label on the pod>" conflicts with pod <the other pod name> that uses the same volume as this pod
+  with SELinuxLabel <the other pod label>. If both pods land on the same node, only one of them may access the volume`.
```

Comment:
```suggestion
  with SELinuxLabel `<the other pod label>`. If both pods land on the same node, only one of them may access the volume`.
```

Reply:
I'm not sure, why only `<the other pod label>` and not e.g. `<label on the pod>`? The whole error message is in backticks / monospace, should `<xxx>` not be monospace?

Reply:
putting that in to backticks is to avoid that the markdown parser treat them as unknown HTML tags.

Reply:
p.s. the parser will render things inside `<` and `>` as `<!-- raw HTML omitted -->`.